vendor/plist-to-html/plist-to-html/PlistToHtml

```diff
@@ -0,0 +1,173 @@
+
+    return {'files': file_sources,
+            'reports': reports}
+
+
+def generate_html_from_plist(file_path, plist, layout_dir, output_path):
+    data = get_report_data_from_plist(plist)
+
+    if data is None or not len(data['reports']):
+        print('No report data in ' + file_path + ' file.')
+        return;
+
+    html_output_path = os.path.join(output_path,
+                                    os.path.basename(file_path) + '.html')
+    html_output = open(html_output_path, 'w+')
+
+    header_path = os.path.join(layout_dir, 'header.html')
+    footer_path = os.path.join(layout_dir, 'footer.html')
+
+    if not os.path.isfile(header_path) or not os.path.isfile(footer_path):
+        print('Failed to open ' + header_path + ' and ' + footer_path)
+        exit(1)
+
+    html_header = open(header_path, 'r')
+    html_footer = open(footer_path, 'r')
+
+    html_output.write(html_header.read())
+
+    # Css files
+    codemirror_css = open(os.path.join(layout_dir, 'codemirror.min.css'), 'r')
+    style_css = open(os.path.join(layout_dir, 'style.css'), 'r')
+    html_output.write('<style type="text/css">')
+    html_output.write(codemirror_css.read() + '\n')
+    html_output.write(style_css.read() + '\n')
+    html_output.write('</style>')
+
+    # Javascript plugins
+    codemirror_js = open(os.path.join(layout_dir, 'codemirror.min.js'), 'r')
+    clike = open(os.path.join(layout_dir, 'clike.min.js'), 'r')
+    bug_viewer = open(os.path.join(layout_dir, 'bugviewer.js'), 'r')
+
+    html_output.write('<script type="text/javascript">')
+    html_output.write(codemirror_js.read() + '\n')
+    html_output.write(clike.read() + '\n')
+    html_output.write(bug_viewer.read() + '\n')
+    html_output.write('var data = ' + json.dumps(data) + ';')
+    html_output.write('</script>')
+
+    html_output.write(html_footer.read())
+
+    print('Html file was generated: ' + html_output_path)
+
+
+def parse(file_path, output_path, layout_dir):
+    """
+    Prints the results in the given file to the standard output in a human-
+    readable format.
+    """
+
+    if not file_path.endswith(".plist"):
+        print("Skipping input file '" + file_path + "' as it is not a plist.")
+        return
+
+    print("Parsing input file '" + file_path + "'")
+
+    try:
+        plist = plistlib.readPlist(file_path)
+        generate_html_from_plist(file_path, plist, layout_dir, output_path)
+    except ExpatError as err:
+        print('Failed to process plist file: ' + file_path +
+              ' wrong file format?', err)
+    except AttributeError as ex:
+        print('Failed to get important report data from plist.', ex)
+    except IndexError as iex:
+        print('Indexing error during processing plist file ' + file_path, iex)
+    except Exception as ex:
+        print('Error during processing reports from the plist file: ' +
+              file_path, ex)
+
+
+def add_arguments_to_parser(parser):
+    parser.add_argument('-i', '--input',
```

Comment:
Wouldn't be enough to name this argument as `input` like here:
https://github.com/Ericsson/codechecker/blob/a3d8f812c236bedfcc89a5cb331156e4a4d66ab4/libcodechecker/libhandlers/store.py#L82-L90

So multiple files/directories can be set and no `-i` argument has to be written in the command line.